mailbox/store/src/test/resources/eml/recursiveMail.json

```diff
@@ -77,38 +77,6 @@
   ],
   "sentDate": "2015-06-10T10:45:27+02:00",
   "attachments": [
-    {
-      "textContent": "Forward as attachment !\n\n\n",
```

Comment:
I'm surprised by this change. If you forward an email as an attachment, it is an attachment, no?

Reply:
```
--------------080603090509090707040003
Content-Type: text/plain; charset=utf-8; format=flowed
Content-Transfer-Encoding: 7bit

Forward as attachment !
```


It's the main body! Not an embeded email. Looks irrelevant to me!

Reply:
yes sorry